docs/user_guide/config_options.md

```diff
@@ -78,6 +78,12 @@ The different running modes of `benchcab` are solely dependent on the options us
 : The `patch` key must be a dictionary like data structure that is compliant with the [`f90nml`][f90nml-github] python package.
 : This key is **optional** and can be omitted from the config file (in which case `patch` does not modify the namelist file).
 
+#### `patch_remove`
+
+: Specifies branch-specific namelist settings to be removed from the `cable.nml` namelist settings. The `patch_remove` key is similar to [`patch`](#`patch`) in the that these settings will be removed from the namelist file for this branch for all science configurations run by `benchcab`.
```

Comment:
> The `patch_remove` key is similar to [`patch`](#`patch`) ...

I don't understand this bit. Are you trying to say they are the opposite of each other?